### DISCUSSION TITLE
Is there a way to tell the validator that _everything_ is required instead of marking every field on every struct as required?

### DISCUSSION THREAD

**Discussion by LordZardeck** [go-playground/validator#1328] [Q&A]
As title says, I'd like to be able to assume everything is required and mark it optional if not required rather than the inverse. I'm writing structs to bind to HTTP request payloads, and the significant majority of the time if i've defined it, it's required.

**Comment by deankarn** [comment]
Unfortunately not, this package is based around exp-licit checks only and is not designed to support implicit ones.
